extensions/binding/org.eclipse.smarthome.binding.hue/README.md

```diff
@@ -102,14 +102,16 @@ Bridge hue:bridge:1 [ ipAddress="192.168.0.64" ] {
 ```
 // Bulb1
 Switch	Light1_Toggle		{ channel="hue:0210:1:bulb1:color" }
-Dimmer  Light1_Dimmer		{ channel="hue:0210:1:bulb1:color" }
-Color 	Light1_Color		{ channel="hue:0210:1:bulb1:color" }
-Dimmer 	Light1_ColorTemp	{ channel="hue:0210:1:bulb1:color_temperature" }
+Dimmer	Light1_Dimmer		{ channel="hue:0210:1:bulb1:color" }
+Color	Light1_Color		{ channel="hue:0210:1:bulb1:color" }
+Dimmer	Light1_ColorTemp	{ channel="hue:0210:1:bulb1:color_temperature" }
+String	Light1_Alert		{ channel="hue:0210:1:bulb1:alert" }
+Switch	Light1_Effect		{ channel="hue:0210:1:bulb1:effect" }
 
 // Bulb2
-Switch	Light2_Toggle		{channel="hue:0220:1:bulb2:brightness"}				
-Dimmer	Light2_Dimm		{channel="hue:0220:1:bulb2:brightness"}
-Dimmer	Light2_ColorTemp	{channel="hue:0220:1:bulb2:color_temperature"}
+Switch	Light2_Toggle		{ channel="hue:0220:1:bulb2:brightness" }
+Dimmer	Light2_Dimm		{ channel="hue:0220:1:bulb2:brightness" }
```

Comment:
The only bad thing about this, is that it renders incorrect if you have configure your tab-width to be 4 chars.